### PR TITLE
Adding new FontFace descriptors

### PR DESCRIPTION
**Pull request by rachelandrew**
I'm trying to make sure we keep the JS APIs for new CSS properties etc. in sync as they tend to get forgotten when I add new stuff in the CSS ref. So this one adds the new FontFace descriptors.

While in there I tidied up the existing pages a bit, added the new spec tables etc.

### CODE REVIEW

**Comment by rachelandrew** [comment]
I'm also sorting out the BCD to add the spec URLs.

File: files/en-us/web/api/fontface/ascentoverride/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - API
+  - Property
+  - Reference
+  - ascentOverride
+  - FontFace
+browser-compat: api.FontFace.ascentOverride
+---
+<div>{{APIRef("CSS Font Loading API")}}</div>
+
+<p class="summary">The <strong><code>ascentOverride</code></strong> property of the {{domxref("FontFace")}} interface returns and sets the value of the {{cssxref("@font-face/ascent-override")}} descriptor. The possible values are <code>normal</code> indicating that the metric used should be obtained from the font file, or a percentage.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/fontface/ascentoverride/index.html:14]:
```suggestion
<p class="summary">The <strong><code>ascentOverride</code></strong> property of the {{domxref("FontFace")}} interface returns and sets the value of the {{cssxref("@font-face/ascent-override")}} descriptor. The possible values are <code>normal</code>, indicating that the metric used should be obtained from the font file, or a percentage.</p>
```

File: files/en-us/web/api/fontface/descentoverride/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - API
+  - Property
+  - Reference
+  - descentOverride
+  - FontFace
+browser-compat: api.FontFace.descentOverride
+---
+<div>{{APIRef("CSS Font Loading API")}}</div>
+
+<p class="summary">The <strong><code>descentOverride</code></strong>  property of the {{domxref("FontFace")}} interface returns and sets the value of the {{cssxref("@font-face/descent-override")}} descriptor. The possible values are <code>normal</code> indicating that the metric used should be obtained from the font file, or a percentage.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/fontface/descentoverride/index.html:14]:
```suggestion
<p class="summary">The <strong><code>descentOverride</code></strong>  property of the {{domxref("FontFace")}} interface returns and sets the value of the {{cssxref("@font-face/descent-override")}} descriptor. The possible values are <code>normal</code>, indicating that the metric used should be obtained from the font file, or a percentage.</p>
```

File: files/en-us/web/api/fontface/linegapoverride/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  - API
+  - Property
+  - Reference
+  - lineGapOverride
+  - FontFace
+browser-compat: api.FontFace.lineGapOverride
+---
+<div>{{APIRef("CSS Font Loading API")}}</div>
+
+<p class="summary">The <strong><code>lineGapOverride</code></strong> property of the {{domxref("FontFace")}} interface returns and sets the value of the {{cssxref("@font-face/line-gap-override")}} descriptor. The possible values are <code>normal</code> indicating that the metric used should be obtained from the font file, or a percentage.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/fontface/linegapoverride/index.html:14]:
```suggestion
<p class="summary">The <strong><code>lineGapOverride</code></strong> property of the {{domxref("FontFace")}} interface returns and sets the value of the {{cssxref("@font-face/line-gap-override")}} descriptor. The possible values are <code>normal</code>, indicating that the metric used should be obtained from the font file, or a percentage.</p>
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**